.github/workflows/simulator.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: make sim-after-import
 #      #  Requires wiring v2
 #      # https://github.com/strangelove-ventures/poa/issues/199
```

Comment:
Will wait for v2 to be merged first

Reply:
[e64d894](https://github.com/strangelove-ventures/poa/pull/204/commits/e64d894cf642bf47c73808fb4f22e8e2e1d2fe10)